django/views/debug.py

```diff
@@ -260,7 +260,7 @@ def get_traceback_data(self):
                     v = pprint(v)
                     # The force_escape filter assumes str, make sure that works
                     if isinstance(v, bytes):
-                        v = v.decode('utf-8', 'replace')  # don't choke on non-utf-8 input
+                        v = v.decode('replace')  # don't choke on non-utf-8 input
```

Comment:
Looks like this block is only used (or at least was only tested) on Python 2. The test added in e0e28bfe715b3f7d4e6cc7ab7bf4000b22c0cf79 doesn't execute it because `v` is `"b'VAL\\xe9VAL'"` (a `str`).